File: Lee.EMS/8.1.0/vue/src/store/entities/action.ts

/*-------------------------------------------------------
// Copyright (C) 2020 Lic
//
// 功能描述:动作管理  实体对象
//
//
// 创建标识: Lee -- 2023-07-30 15:32:44
//
//------------------------------------------------------*/
 
import Entity from './entity'
// 动作管理 
export default class Action extends Entity<number>{
    moduleid:number;
    name:string;
    icon:string;
    sort:number;
    visible:string;
    methodCode:string;
}

File: Lee.EMS/8.1.0/aspnet-core/src/Lee.EMS.Core/SYS/Action.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Abp.Domain.Entities.Auditing;
using System.Diagnostics.CodeAnalysis;

namespace Lee.EMS.SYS
{
    [Table("Sys_Action")]
    public  class Action : AuditedEntity<long>
    {
        /// <summary>
        /// 
        /// </summary>
        [Column("ModuleId"), NotNull]
        public long ModuleId { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("Name"), MaxLength(100), NotNull]
        public string Name { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("Icon"), MaxLength(200)]
        public string Icon { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("Sort")]
        public long Sort { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("Visible"), MaxLength(1)]
        public string Visible { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("MethodCode"), MaxLength(100)]
        public string MethodCode { get; set; }
    }
}
